bench.Dockerfile

```diff
@@ -1,16 +1,17 @@
 FROM debian:bullseye
 RUN apt-get update -qq && apt-get install -qq -yy make curl gcc g++ patch bzip2 git unzip
 RUN adduser --disabled-password --gecos '' --shell /bin/bash opam
+ENV OPAMREPOSHA 26770281fa1ea8b13aab979c1dfbd326e9ab512c
+RUN git clone https://github.com/ocaml/opam-repository --depth 1 /rep/opam-repository
+RUN git -C /rep/opam-repository fetch origin $OPAMREPOSHA
+RUN git -C /rep/opam-repository checkout $OPAMREPOSHA
 USER opam
 COPY --chown=opam:opam . /src
 WORKDIR /src
 RUN make cold
 USER root
 RUN install ./opam /usr/local/bin/
 USER opam
-ENV OPAMREPOSHA 26770281fa1ea8b13aab979c1dfbd326e9ab512c
-RUN git clone https://github.com/ocaml/opam-repository --depth 1
-RUN git -C opam-repository fetch origin $OPAMREPOSHA
-RUN git -C opam-repository checkout $OPAMREPOSHA
-RUN opam init -n --disable-sandboxing ./opam-repository
-RUN find "$(pwd)/opam-repository" -name opam -type f > /home/opam/all-opam-files
+RUN opam init --bare -n --disable-sandboxing /rep/opam-repository
+RUN opam switch create --fake default 4.14.0
+RUN find /rep/opam-repository -name opam -type f > /home/opam/all-opam-files
```

Comment:
Reading file bench test depends on `OPAMREPOSHA`, that can evolve in the future.

Reply:
Sure but we'd see that evolution on the graph associated with the commit changing it so i don't think this is an issue.

Reply:
The idea of these benches is to have them stable over the time, to only see changes related to test target, no?

Reply:
stable over runs yes, not necessarily stable over time, i would say. Benchmark stable over time is an extremely sticky thing to do and i don't think we should meddle with that (e.g. at some point in the future the machine running those benchmarks will die and the result of the bench will change)

Reply:
Of course we can't rely on it indefinitely, as it depends on a machine, but i think that we shouldn't have a benchmark that changes if an external value changes (here repo hash). In fact, it is the same for other opam install bench tests too, they depend son that repo hash too.
Anyway, in the future if we need to change that hash, we can update the test accordingly (e.g. have 2 repo hashes).